appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java

```diff
@@ -97,6 +97,13 @@ protected void onLoad() {
         int idx = 0;
         int width = 320;
         int height = 505;
+
+        if (val.equals("False")) {
```

Comment:
Should be "True", not "False". true is tablet, false is phone.

Reply:
done